packages/daemon/test/test-endo.js

```diff
@@ -354,6 +354,21 @@ test('store with name', async t => {
   }
 });
 
+test('move renames value', async t => {
```

Comment:
Would enjoy tests that cover moving to and from other hubs. Should verify that we fall through to `rename` only if the to and from hubs fulfill to identical values and otherwise _carefully_ add _then_ delete, so that failure leaves the name intact at the source. And a test for failure to add to a bogus target hub.